server/controllers/profile-controller.js

```diff
@@ -0,0 +1,71 @@
+import { successResponse, errorResponse } from '../utils/helpers';
+import db from '../models';
+
+const { User } = db;
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
```

Comment:
Again please be more careful with documentation 🙂

Reply:
ok I will work on this